web-nfc/idlharness.https.window.js

```diff
@@ -19,6 +23,8 @@ idl_test(
     idl_array.add_objects({
       NFCWriter: ['new NFCWriter();'],
       NFCReader: ['new NFCReader();'],
+      NDEFRecord: [`new NDEFRecord(${JSON.stringify(record)});`],
+      NDEFMessage: [`new NDEFMessage([${JSON.stringify(message)}]);`],
```

Comment:
No [] for embracing ${JSON.stringify(message)}

Reply:
Done. 😳